backend/routes/PostReaction/CommentHelperFunc.py

```
@@ -0,0 +1,26 @@
+from fastapi import APIRouter, Depends, HTTPException, Query
```

Ruff (0.8.2)

`fastapi.APIRouter` imported but unused

Remove unused import

(F401)